patches/chrome-common-importer-profile_import_process_param_traits_macros.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/common/importer/profile_import_process_param_traits_macros.h b/chrome/common/importer/profile_import_process_param_traits_macros.h
+index 81d070e74f0a095801dde6a96967011eb2131c41..ecc097f560ea0f40aa270e5d3dd516ad6d344797 100644
+--- a/chrome/common/importer/profile_import_process_param_traits_macros.h
++++ b/chrome/common/importer/profile_import_process_param_traits_macros.h
+@@ -24,7 +24,7 @@ IPC_ENUM_TRAITS_MIN_MAX_VALUE(importer::ImporterType,
+ #else
+ IPC_ENUM_TRAITS_MIN_MAX_VALUE(importer::ImporterType,
+                               importer::TYPE_UNKNOWN,
+-                              importer::TYPE_BOOKMARKS_FILE)
```

Comment:
Can this be done via a `chromium_src` override? Maybe `#define TYPE_BOOKMARKS_FILE TYPE_EDGE`?

Reply:
done